scopus-import/src/main/java/no/sikt/nva/scopus/conversion/ContributorExtractor.java

```diff
@@ -277,9 +301,14 @@ private Organization generateOrganizationWithLabel(Map<String, String> label) {
     }
 
     private Optional<Map<String, String>> getOrganizationLabels(AuthorGroupTp authorGroup) {
-        var organizationNameOptional = getOrganizationNameFromAuthorGroup(authorGroup);
-        return organizationNameOptional.map(
-            organizationName -> Map.of(getLanguageIso6391Code(organizationName), organizationName));
+        var organizationName = getOrganizationNameFromAuthorGroup(authorGroup);
+        if (organizationName.isPresent() && !organizationName.get().isEmpty()) {
```

Comment:
```
    return isKnownOrganization(organizationName) 
            ? organizationName.map(name -> Map.of(getLanguageIso6391Code(name), name))
            : someClearMethodName(authorGroup);
```

Reply:
Not sure about the naming here, nor what is going on…

Reply:
extract to extractCountryNameAsLabel